[NR-65072] The chart readme was not covering the correct default (#159)
newrelic/newrelic-prometheus-configurator: README.md

For further information of the configuration needed for the chart just read the [chart's README](/charts/newrelic-prometheus-agent/README.md).

## Getting Started
The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster.

> All pod and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.

Reviewer: ```suggestion
All pods and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.
```

Moreover, the solution will scrape as well all pods and endpoints with the `prometheus.io/scrape: true` annotations and

> having one of the labels matching the integrations_filter configuration.

Reviewer: ```suggestion
have one of the labels matching the integrations_filter configuration.
```

Author: 🤔 I believe in this case it is correct. 

it would be have if it was " and `that` have one of the..."

Notice that at any point you can turn off the integrations filters and scrape all pods and services annotated with
`prometheus.io/scrape: true` by setting `config.kubernetes.integrations_filter.integrations_filter: false` or turning
it off in any specific job.

### Usage

> For further information checkout [the official documentation](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) 

Reviewer: ```suggestion
For further information check out [the official documentation](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) 
```

Author: oh, you are right https://dictionary.cambridge.org/dictionary/english/checkout
I think I have been always using it as a verb 😖
